### ISSUE TITLE
Error and general issues with the instructions

### ISSUE THREAD

**Issue by OPCIT** [govCMS/audit-site#39]
Instructions are very light on for someone new to this.

Also running command:

wget https://github.com/govCMS/audit-site/releases/download/7.x-3.x-beta2/audit.phar

results in the following:

--2019-10-24 13:20:55--  https://github.com/govCMS/audit-site/releases/download/7.x-3.x-beta2/audit.phar
Resolving github.com (github.com)... 13.237.44.5
Connecting to github.com (github.com)|13.237.44.5|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-10-24 13:20:55 ERROR 404: Not Found.

As this is an integral part of the forklift process are there plans to fix this and make it easiest to carry out?

**Comment by fubarhouse** [comment]
@OPCIT,

I've rectified this, the response code now returns 200 and it's good to download, the `phar` should have previously been created for the last two releases, but it wasn't. I will make a conservative effort to make sure this doesn't happen again.

**Comment by OPCIT** [comment]
Follow up to issues that continue to exist.
Running script still returns a number of false negatives, specifically around the:

CSS aggregation is enabled, says it's not when it is.
Javascript aggregation: says it's not when it is.
Hide errors from screen; says there are not hidden when they are.

When will the latest audit.site code be released that rectifies these issues?

**Comment by tobybellwood** [comment]
The issue here (assuming Drupal8) may be that the development and production settings.php have config overrides in place for errors, js and css (https://github.com/govCMS/govcms8lagoon/blob/develop/.docker/images/govcms8/settings/development.settings.php) - audit-site is currently not able to detect when a config override is in place, and most likely tests against the value in config, not the computed value.

**Comment by fubarhouse** [comment]
A static file check could potentially look for global overrides. We’ve not explored that idea before though.

**Comment by OPCIT** [comment]
Yes a D8 site. We obviously have no access to the settings.php so I assume we continue to mark the site audit report off as a false negative for these elements, until such time that it's fixed.

 govcms8lagoon/.docker/images/govcms8/settings/development.settings.php 

/**
 * Set expiration of cached pages to 0.
 */
$config['system.performance']['cache']['page']['max_age'] = 0;

/**
 * Disable CSS and JS aggregation.
 */
$config['system.performance']['css']['preprocess'] = FALSE;
$config['system.performance']['js']['preprocess'] = FALSE;
